pipelines/frontend-pipeline.yml

```diff
@@ -0,0 +1,83 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - frontend/*
+      - pipelines/frontend-pipelines.yml
+    
+  pr:
+    autoCancel: true   
+    branches:
+      include:
+        - master
+    paths:
+      include:
+        - frontend/*
+        - pipelines/frontend-pipelines.yml
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  - group: prodoh-urlist-variable-group
+
+stages:
+  - stage: BuildStage
+    displayName: Build VUE Frontend
+    jobs:
+      - job: BuildJob
+        displayName: Build Test and Publish
+        steps:
+          - task: Npm@1
+            displayName: setup Npm
+            inputs:
+              command: 'install'
+              workingDir: './'
```

Comment:
Wouldn't the working dir be `./frontend` ?

Reply:
good catch, my local repo has the frontend only..will update